$NetBSD$

Use libhidapi from pkgsrc.
Treat the BSDs like Linux for wxWidgets.

--- src/CMakeLists.txt.orig	2024-06-15 05:46:24.368814750 +0000
+++ src/CMakeLists.txt
@@ -32,8 +32,6 @@ endif ()
 if (SLIC3R_GUI)
     add_subdirectory(imgui)
     add_subdirectory(libvgcode)
-    add_subdirectory(hidapi)
-    include_directories(hidapi/include)
 
     if(WIN32)
         message(STATUS "WXWIN environment set to: $ENV{WXWIN}")
@@ -46,7 +44,7 @@ if (SLIC3R_GUI)
         endif()
     endif()
 
-    if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    if (CMAKE_HOST_LINUX OR CMAKE_HOST_BSD)
         set (wxWidgets_CONFIG_OPTIONS "--toolkit=gtk${SLIC3R_GTK}")
     endif ()
     find_package(wxWidgets 3.2 MODULE REQUIRED COMPONENTS base core adv html gl webview)
